internal/runbits/errors/errors.go

```diff
@@ -82,6 +82,20 @@ func (o *OutputError) MarshalOutput(f output.Format) interface{} {
 	return strings.Join(outLines, "\n")
 }
 
+func formatMessage(message string) []string {
```

Comment:
Please add a brief comment about what this function does. The name is not specific enough.